runtime/criusupport/criusupport.cpp

```diff
@@ -435,6 +435,7 @@ Java_org_eclipse_openj9_criu_CRIUSupport_checkpointJVMImpl(JNIEnv *env,
 			goto wakeJavaThreadsWithExclusiveVMAccess;
 		}
 		if (vm->portLibrary->checkpointRestoreTimeDelta < 0) {
+			systemReturnCode = (IDATA)vm->portLibrary->checkpointRestoreTimeDelta;
```

Comment:
This loses information (casting from `int64_t` to `IDATA` - which may only be 32 bits).

Reply:
`A negative value calculated for checkpointRestoreTimeDelta` message is sufficient for such scenario, `Trc_CRIU_before_checkpoint` & `Trc_CRIU_after_checkpoint` can be used for further investigation.

Reply:
Added a comment.